source/features/avoid-accidental-submissions.tsx

```diff
@@ -9,32 +10,48 @@ function addQuickSubmit(): void {
 }
 
 function onKeyDown(event: delegate.Event<KeyboardEvent, HTMLInputElement>): void {
+	const form = event.delegateTarget.form!;
 	if (
 		event.key !== 'Enter'
 		|| event.ctrlKey
 		|| event.metaKey
 		|| event.isComposing // #4323
-		|| select.exists('.suggester', event.delegateTarget.form!) // GitHub’s autocomplete dropdown
+		|| select.exists([
+			'.suggester', // GitHub’s autocomplete dropdown
+			'.rgh-avoid-accidental-submissions.my-1',
+		], form)
 	) {
 		return;
 	}
 
+	if (!select.exists('.btn-primary[type="submit"]:not(:disabled)', form)) {
```

Comment:
Does this also work? It needs to be tested in every form, because it's not exactly equivalent.
```suggestion
	if (select.exists('.btn-primary[type="submit"]:disabled', form)) {
```

Reply:
Yes, I tested it and it works in every form.